sdk/python/feast/feature_store.py

```diff
@@ -1690,6 +1690,67 @@ def _get_online_features(
         )
         return OnlineResponse(online_features_response)
 
+    @log_exceptions_and_usage
+    def retrieve_online_documents(
```

Comment:
There's probably something to be said about having a configurable distance metric to let the user choose which way to get the `top_k`